docs/source/NEWS.rst

```diff
@@ -1,6 +1,18 @@
 Changelog
 =========
 
+Unreleased 
```

Comment:
We are changed NEWS.rst today by adding unreleased 16.3.0 version. Could you please merge current `master` branch of txmongo into your branch and resolve the merge conflict?